### ISSUE TITLE
Collection.get is missing offset and include options

### ISSUE THREAD

**Issue by Jegorex** [amikos-tech/chromadb-java-client#21]
According to the Chroma documentation there is an offset and include value for the get method.

These seem to be missing?

**Comment by tazarov** [comment]
You are right @Jegorex, let me add it over the weekend, unless you already have a fix ready.

**Comment by Jegorex** [comment]
Thank you.
I don't have a fix for it, but noticed it while looking through the documentation.

**Comment by oss-amikos** [comment]
Superseded by #84 (record operation builders with full get support including offset, limit, include). See tracking issue #94.
